{"name": "Invalid credentials", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//input[@type='email']\"}\n  (Session info: chrome=114.0.5735.198); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x00000001012dbf48 chromedriver + 4226888\n1   chromedriver                        0x00000001012d44f4 chromedriver + 4195572\n2   chromedriver                        0x0000000100f18d68 chromedriver + 281960\n3   chromedriver                        0x0000000100f53fc8 chromedriver + 524232\n4   chromedriver                        0x0000000100f8ac58 chromedriver + 748632\n5   chromedriver                        0x0000000100f47f1c chromedriver + 474908\n6   chromedriver                        0x0000000100f48ef4 chromedriver + 478964\n7   chromedriver                        0x000000010129d59c chromedriver + 3970460\n8   chromedriver                        0x00000001012a16f0 chromedriver + 3987184\n9   chromedriver                        0x00000001012a75b4 chromedriver + 4011444\n10  chromedriver                        0x00000001012a22fc chromedriver + 3990268\n11  chromedriver                        0x000000010127a1c0 chromedriver + 3826112\n12  chromedriver                        0x00000001012be088 chromedriver + 4104328\n13  chromedriver                        0x00000001012be1e0 chromedriver + 4104672\n14  chromedriver                        0x00000001012cdf28 chromedriver + 4169512\n15  libsystem_pthread.dylib             0x00000001890e7fa8 _pthread_start + 148\n16  libsystem_pthread.dylib             0x00000001890e2da0 thread_start + 8\n\n", "trace": "  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login.py\", line 32, in step_impl\n    email_ele = CommonMethods().find_element(context.driver, email_field)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/Helpers/common_methods.py\", line 11, in find_element\n    return driver.find_element(list(locator.keys())[0], list(locator.values())[0])\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "When user enters invalid username and password", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//input[@type='email']\"}\n  (Session info: chrome=114.0.5735.198); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x00000001012dbf48 chromedriver + 4226888\n1   chromedriver                        0x00000001012d44f4 chromedriver + 4195572\n2   chromedriver                        0x0000000100f18d68 chromedriver + 281960\n3   chromedriver                        0x0000000100f53fc8 chromedriver + 524232\n4   chromedriver                        0x0000000100f8ac58 chromedriver + 748632\n5   chromedriver                        0x0000000100f47f1c chromedriver + 474908\n6   chromedriver                        0x0000000100f48ef4 chromedriver + 478964\n7   chromedriver                        0x000000010129d59c chromedriver + 3970460\n8   chromedriver                        0x00000001012a16f0 chromedriver + 3987184\n9   chromedriver                        0x00000001012a75b4 chromedriver + 4011444\n10  chromedriver                        0x00000001012a22fc chromedriver + 3990268\n11  chromedriver                        0x000000010127a1c0 chromedriver + 3826112\n12  chromedriver                        0x00000001012be088 chromedriver + 4104328\n13  chromedriver                        0x00000001012be1e0 chromedriver + 4104672\n14  chromedriver                        0x00000001012cdf28 chromedriver + 4169512\n15  libsystem_pthread.dylib             0x00000001890e7fa8 _pthread_start + 148\n16  libsystem_pthread.dylib             0x00000001890e2da0 thread_start + 8\n\n", "trace": "  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login.py\", line 32, in step_impl\n    email_ele = CommonMethods().find_element(context.driver, email_field)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/Helpers/common_methods.py\", line 11, in find_element\n    return driver.find_element(list(locator.keys())[0], list(locator.values())[0])\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1688920390355, "stop": 1688920395376}, {"name": "Then login should fail", "status": "skipped", "start": 1688920395376, "stop": 1688920395376}], "start": 1688920390355, "stop": 1688920395376, "uuid": "1d75bd62-c7ac-49c0-9416-d26fa640c94d", "historyId": "89a048edcbc8248da26faa31dabfcee1", "fullName": "Check login feature: Invalid credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "smoke"}, {"name": "feature", "value": "Check login feature"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}